etc/inc/firefox-common-addons.inc

```diff
@@ -75,16 +83,7 @@ include allow-python3.inc
 
 # ff2mpv
 #ignore noexec ${HOME}
-#noblacklist ${HOME}/.config/mpv
-#noblacklist ${HOME}/.config/youtube-dl
-#noblacklist ${HOME}/.netrc
 #include allow-lua.inc
-#include allow-python3.inc
-#mkdir ${HOME}/.config/mpv
-#mkdir ${HOME}/.config/youtube-dl
-#whitelist ${HOME}/.config/mpv
-#whitelist ${HOME}/.config/youtube-dl
-#whitelist ${HOME}/.netrc
 #whitelist /usr/share/lua
 #whitelist /usr/share/lua*
 #whitelist /usr/share/vulkan
```

Comment:
mpv does not work properly without this wusc additions (depending on configuration and system) IIRC. IHMO we should enable it too.

Reply:
you mean the /usr lines?